grammar.js

```diff
@@ -92,7 +147,19 @@ module.exports = grammar({
                 field("denominators", repeat(seq(/ ?\//, $._units)))
             ),
         _units: ($) => seq($.unit, repeat(seq(/ ?\./, $.unit))),
-        unit: ($) => seq(token.immediate(unit), optional($.exposant)),
-        exposant: () => token.immediate(choice("²", "³")),
+        unit: ($) => seq(unit, optional($.exposant)),
+        exposant: () => /²|³/,
```

Comment:
Je me balade ici, les autres exposants existent aussi : ⁰, ¹, ⁵, etc. Les codes unicodes sont de U+2070 à U+2079:

```suggestion
        exposant: () => /[\u{2070}-\u{2079}]+/u,
```

Reply:
Haha merci :p
On pourrait faire une petite séance de pair un de ces quatre si le sujet t'interesse !